src/main/resources/com/mathworks/ci/RunMatlabTestsStep/help-selectByFolder.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    <p>Specify the location of folders containing test code, relative to the project root folder, as a string array. The specified folders and their subfolders are added to the test suite runner.</p>
+    <p><b>Example:</b> ["test/unitTest", "test/IntegTests"]</p>
```

Comment:
Specify the locations of folders containing tests, relative to the project root folder, as a string array.  To generate a test suite, MATLAB uses only the tests in the specified folders and their subfolders.

<p><b>Example:</b> ['test/folderA', 'test/folderB']</p>

Reply:
Just changed String array to List of string